recipes/pcdsdevices/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pcdsdevices" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/pcdsdevices/archive/v{{ version }}.tar.gz
+  sha256: 8949526819a6c837e684d9e77de738adb013cedd98a26495e8bc1ef063f0546d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bluesky >=1.5.0
+    - cf-units >=2.0.2
+    - happi
+    - ophyd >=1.4.0
+    - pyepics
+    - pyyaml
+    - scipy
+
+test:
+  imports:
+    - pcdsdevices
+
+about:
+  home: https://github.com/pcdshub/pcdsdevices
+  doc_url: https://pcdshub.github.io/pcdsdevices/
+  dev_url: https://github.com/pcdshub/pcdsdevices
+  license: SLAC Open
```

Comment:
```suggestion
  license: LicenseRef-SLAC
```

Reply:
The linter didn't like this change. Can we switch back to `SLAC Open` and proceed?

Reply:
Can you open an issue in conda-smithy repo?

Reply:
Done https://github.com/conda-forge/conda-smithy/issues/1272

Reply:
This license is BSD-3-Clause-LBNL right? https://spdx.org/licenses/BSD-3-Clause-LBNL.html

Reply:
No... https://github.com/pcdshub/pcdsdevices/blob/master/LICENSE.md

Reply:
Only the copyright changed here right? See section 2.1.3 of https://spdx.org/spdx-license-list/matching-guidelines

cc @scopatz 

Btw, the SPDX recommendation by the linter is new.

Reply:
@isuruf at a first glance it looks like it is just the copyright. Should we add in this case just BSD and move on?

Reply:
I think we should go with either `BSD-3-Clause-LBNL` or `LicenseRef-BSD-3-Clause-SLAC`. If the former, there's no warnings. If the latter, there'll be a warning, but we can ignore that and merge and fix the linter later.

Reply:
Let's do ` LicenseRef-BSD-3-Clause-SLAC`. 👍

Reply:
I also opened https://github.com/spdx/license-list-XML/issues/994